src/plugins/tables/_base.scss

```diff
@@ -189,6 +189,18 @@ table {
 				}
 			}
 
+			button {
+				&.sorting-cnt {
+					-moz-appearance: none;
+					-webkit-appearance: none;
+					background: transparent;
+					border: 0;
+					font-family: inherit;
+					font-size: 100%;
```

Comment:
Is this needed? Is there a value added to declare a font-size to 100%?

Reply:
After testing, it seems that removing  "font-size: 100%" as no effect on the content.